tests/test_emulator.py

```diff
@@ -0,0 +1,8 @@
+def test_emulator_install():
+    from py21cmemu import Emulator
+    emu = Emulator()
+
+def test_emulator_runs():
+    from py21cmemu import Emulator
```

Comment:
I don't think you need these tests -- these tests should be just in the py21cmemu package (which they are).

Reply:
Just to make sure that 21cmEMU is installed properly e.g. they were failing before when I had the weird tensorflow bug. The tests of py21cmemu are not ran upon install, so I thought this would be useful to make sure it's installed properly. But I can remove them if you think it's not necessary.

Reply:
Yeah, I think having the other test -- of the likelihood itself -- is enough to show that the package is installed.